platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateMap.java

```diff
@@ -47,24 +52,55 @@ public SignedStateMap() {
      * @return an auto-closable object that wraps a signed state. May point to a null state if there is no state for the
      * given round. Will automatically release the state when closed.
      */
-    public synchronized AutoCloseableWrapper<SignedState> get(final long round) {
+    public synchronized @NonNull AutoCloseableWrapper<SignedState> get(final long round) {
+        return newSignedStateWrapper(map.get(round));
+    }
 
-        final SignedState signedState = map.get(round);
+    /**
+     * Get the latest state in this map. A reservation is taken on the state before this method returns.
+     *
+     * @return an auto-closable object that wraps a signed state. May point to a null state if there are no states in
+     * this container. Will automatically release the state when closed.
+     */
+    public synchronized @NonNull AutoCloseableWrapper<SignedState> getLatest() {
+        if (map.isEmpty()) {
+            return newSignedStateWrapper(null);
+        }
+        return newSignedStateWrapper(map.get(map.lastKey()));
+    }
+
+    /**
+     * Get the latest round in this map.
+     *
+     * @return the latest round in this map, or {@link #NO_STATE_ROUND} if this map is empty
+     */
+    public synchronized long getLatestRound() {
+        if (map.isEmpty()) {
+            return NO_STATE_ROUND;
+        }
+        return map.lastKey();
+    }
 
-        return newSignedStateWrapper(signedState);
+    /**
+     * Check if the map is empty.
+     *
+     * @return true if the map is empty, otherwise false.
+     */
+    public synchronized boolean isEmpty() {
+        return map.isEmpty();
     }
 
     /**
      * Take a reservation.
      */
-    private static void reserve(final SignedState signedState) {
+    private static void reserve(@NonNull final SignedState signedState) {
```

Comment:
what's the point of these private static methods `reserve` and `release`? Why not just call the `SignedState` methods directly?